### DISCUSSION TITLE
Tabellen-Modul: Edit-Mode sollte verbessert werden, um dem Nutzer eine flexiblere Handhabung mit Spalten und Zeilen zu ermöglichen

### DISCUSSION THREAD

**Discussion by techotline** [lotta-schule/web#234] [Ideen]
Eine Spalte oder Zeile, die im Beitrag vorhanden ist, muss zuerst gelöscht und der Inhalt erneut hinzugefügt werden, um ein chronologisches Leseverhalten zu gewährleisten. Daher schlage ich vor, den Edit-Modus für Autoren zu verbessern und das Modul mit einem "Verschieben"-Tool auszustatten. Das Prinzip (Drag and Drop**) könnte wie bei allen Modulen sein, wenn sie innerhalb eines Beitrags an die neue gewünschte Position bewegt werden möchten. 

**
<img width="757" alt="dragbar" src="https://github.com/lotta-schule/web/assets/153733311/6e7f9b0d-c25b-4afe-99d1-7962ca7cf1e9">

Das wiederum könnte den Prozess vereinfachen, um schneller und effektiver Inhalte zu aktualisieren.

**Comment by ptitmouton** [comment]
Wir machen viele kleine Tickets draus. Bitte überlegt euch ganz genau was möglich sein soll

**Reply by techotline** [reply]
- Zeile / Spalte verschieben
- beliebige (jede!) Spalte / Zeile löschen können

**Reply by wie0815** [reply]
Das wäre ein guter Anfang!

**Reply by wie0815** [reply]
In dem Kontext mach es wahrscheinlich Sinn, wenn man:
- an beliebiger Stelle Spalten/Zeilen hinzufügen könnte!
